lib/main.dart

```diff
@@ -172,10 +209,12 @@ class _CardInterfaceState extends State<CardInterface> {
         width: MediaQuery.of(context).size.width / 2.5,
         height: MediaQuery.of(context).size.height / 1.25,
         child: Card(
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          child: ListView(
```

Comment:
Why did we change to `ListView`? I thought we weren't making this form as a List of components, which is why I think we got rid of ListTile usage.